### PR TITLE
Expand repeated annotations in libraries

### PR DESCRIPTION
**Pull request by ting-yuan**
they were presented as containers

### CODE REVIEW

**Review by dx404 — APPROVED** [review]

**Comment by ting-yuan** [comment]
Fixes #1883

File: kotlin-analysis-api/testData/annotationsRepeatable.kt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+// FILE: lib/A.java
+package lib;
+import java.lang.annotation.Repeatable;
+@Repeatable(A.C.class)
+    @interface A {
+        int value();
+        @interface C {
+            A[] value();
+        }
+    }
```
Comment (kuanyingchou) [inline kotlin-analysis-api/testData/annotationsRepeatable.kt:50]:
```suggestion
@interface A {
    int value();
    @interface C {
        A[] value();
    }
}
```

File: kotlin-analysis-api/testData/annotationsRepeatable.kt [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+// FILE: lib/KA.kt
+    package lib
```
Comment (kuanyingchou) [inline kotlin-analysis-api/testData/annotationsRepeatable.kt:52]:
```suggestion
package lib
```

File: test-utils/testData/api/annotationsRepeatable.kt [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+// FILE: lib/A.java
+package lib;
+import java.lang.annotation.Repeatable;
+@Repeatable(A.C.class)
+    @interface A {
+        int value();
+        @interface C {
+            A[] value();
+        }
+    }
```
Comment (kuanyingchou) [inline test-utils/testData/api/annotationsRepeatable.kt:47]:
```suggestion
@interface A {
    int value();
    @interface C {
        A[] value();
    }
}
```

File: test-utils/testData/api/annotationsRepeatable.kt [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+// FILE: lib/KA.kt
+    package lib
```
Comment (kuanyingchou) [inline test-utils/testData/api/annotationsRepeatable.kt:49]:
```suggestion
package lib
```

**Review by kuanyingchou — APPROVED** [review]
Thanks!

**Pull request merged**